berserk/clients/teams.py

```diff
@@ -50,3 +50,49 @@ def kick_member(self, team_id: str, user_id: str) -> None:
         """
         path = f"/team/{team_id}/kick/{user_id}"
         self._r.post(path)
+
+    def get_join_request(
+        self, team_id: str, declined: bool = False
+    ) -> Iterator[Dict[str, Any]]:
+        """Get pending join requests of your team
+
+        :param team_id: ID of a team
+        :param declined: Get the declined join requests
+        :return: Iterator over join requests
+        """
+        path = f"/api/team/{team_id}/requests"
+        params = {"declined": declined}
+        yield from self._r.get(
+            path,
+            params=params,
+            stream=True,
+        )
+
+    def accept_join_request(self, team_id: str, user_id: str) -> Dict[str, Any]:
+        """Accept someone's request to join the team
+
+        :param team_id: ID of a team
+        :param user_id: ID of a member requesting to join
+        :return: Dictionary informing if accepted or rejected
+        """
+        path = f"/api/team/{team_id}/request/{user_id}/accept"
+        return self._r.post(path)
+
+    def get_swiss_tournament(
```

Comment:
This endpoint is already implemented via `client.tournaments.swiss_by_team`. If you want to reexport here I guess you could do `get_swisses = Tournaments.swiss_by_team`.

Reply:
I am removing this end point all together. This require creating an object for tournaments to call it.

Reply:
That's actually an interesting consideration since the endpoint is grouped below "swiss tournaments" in the API docs but the URL is `/api/team`. Even worse, we only have a "tournament" client that combines swiss and arena.

But I think generally, I prefer grouping things similar to the API docs (the URLs feel more like an implementation detail) and I'd rather not duplicate endpoints since it can be quite confusing.